packages/puik/vite.config.ts

```diff
@@ -17,7 +17,8 @@ export default defineConfig({
       external: [
         ...Object.keys(pkg.dependencies),
         ...Object.keys(pkg.peerDependencies),
-        /^@prestashopcorp\/puik-components\/.*/
+        /^@prestashopcorp\/puik-components\/.*/,
+        /^@prestashopcorp\/puik-web-components\/.*/
```

Comment:
```suggestion
```
If the web-components aren't imported in this package you can remove this line (using a relative path to the component file)